packages/run-protocol/scripts/init-core.js

```diff
@@ -1,68 +1,169 @@
 /* global process */
+// @ts-check
 import { makeHelpers } from '@agoric/deploy-script-support';
 
+import {
+  getManifestForRunProtocol,
+  getManifestForEconCommittee,
+  getManifestForMain,
+  getManifestForRunStake,
+  getManifestForPSM,
+} from '../src/core-proposal.js';
+
+/** @type {Record<string, Record<string, [string, string]>>} */
+const installKeyGroups = {
+  econCommittee: {
+    contractGovernor: [
+      '@agoric/governance/src/contractGovernor.js',
+      '../bundles/bundle-contractGovernor.js',
+    ],
+    committee: [
+      '@agoric/governance/src/committee.js',
+      '../bundles/bundle-committee.js',
+    ],
+    binaryVoteCounter: [
+      '@agoric/governance/src/binaryVoteCounter.js',
+      '../bundles/bundle-binaryVoteCounter.js',
+    ],
+  },
+  runStake: {
+    runStake: ['../src/runStake/runStake.js', '../bundles/bundle-runStake.js'],
+  },
+  main: {
+    amm: [
+      '../src/vpool-xyk-amm/multipoolMarketMaker.js',
+      '../bundles/bundle-amm.js',
+    ],
+    vaultFactory: [
+      '../src/vaultFactory/vaultFactory.js',
+      '../bundles/bundle-vaultFactory.js',
+    ],
+    liquidate: [
+      '../src/vaultFactory/liquidateMinimum.js',
+      '../bundles/bundle-liquidateMinimum.js',
+    ],
+    reserve: ['../src/reserve/assetReserve.js', '../bundles/bundle-reserve.js'],
+  },
+  psm: {
+    psm: ['../src/psm/psm.js', '../bundles/bundle-psm.js'],
+  },
+};
+
+const { entries, fromEntries } = Object;
+
+/** @type { <K extends string, T, U>(obj: Record<K, T>, f: (t: T) => U) => Record<K, U>} */
+const mapValues = (obj, f) =>
+  // @ts-ignore entries() loses the K type
+  harden(fromEntries(entries(obj).map(([p, v]) => [p, f(v)])));
+
+const committeeProposalBuilder = async ({ publishRef, install }) => {
+  const { ROLE = 'chain' } = process.env;
+
+  // preload ERTP, marshal, store, etc.
+  const [mod0, bundle0] = installKeyGroups.econCommittee.binaryVoteCounter;
+  await install(mod0, bundle0, { persist: true });
```

Comment:
Would this create the first of two separate Zoe installations?  Should the below `publishGroup` avoid doing an install for `group === 'econCommittee' && mod === 'binaryVoteCounter'`?

Reply:
Yes, it makes a throw-away installation, but I didn't think it was worth fixing. Will do.